README.md

```diff
@@ -14,22 +14,22 @@ FBT is an internationalization framework designed to be both powerful and flexib
 
 ## Building fbt
 ```
-git clone git@github.com:facebookincubator/fbt.git;
-cd fbt;
-yarn install;
+git clone https://github.com/facebookincubator/fbt
```

Comment:
Any reason for avoiding SSH on Windows?  If not, can we just stick to the semicolon removal?

Reply:
No particular reason, just a gut feeling change as the git over https is (from personal experience) usually preferred some reason with READMEs on GitHub even on FB repos, such as in 
https://github.com/facebook/create-react-app/blob/0702e78fbbf8f481d04255dd76f525ac6cfef0b9/CONTRIBUTING.md#setting-up-a-local-copy

https://github.com/facebookincubator/offline-conversion-file-uploader#install

https://github.com/facebookincubator/Bowler/blob/2cc74be49c1d2d53ff4b46045a34dfaab50b9263/CONTRIBUTING.md#L15

and others :) 

Anyway reverted that in https://github.com/facebookincubator/fbt/pull/12/commits/e170e71ba7a65571f800b18d8bca409abbae6502

Reply:
Happy to take the change after I suppose. I'll pedantically claim "It's better to keep the fix separate"